# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libsoup/libsoup_gnutls.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -r -u -N libsoup-2.2.2.orig/libsoup/soup-gnutls.c libsoup-2.2.2/libsoup/soup-gnutls.c
--- libsoup-2.2.2.orig/libsoup/soup-gnutls.c	2005-03-28 02:58:00.256716192 +0200
+++ libsoup-2.2.2/libsoup/soup-gnutls.c	2005-03-28 02:59:15.378295976 +0200
@@ -63,7 +63,6 @@
 	}
 
 	if (status & GNUTLS_CERT_INVALID ||
-	    status & GNUTLS_CERT_NOT_TRUSTED ||
 	    status & GNUTLS_CERT_REVOKED)
 	{
 		g_set_error (err, SOUP_SSL_ERROR,
